staging/src/k8s.io/apiserver/pkg/apis/config/types.go

```diff
@@ -89,4 +89,13 @@ type KMSConfiguration struct {
 	// Timeout for gRPC calls to kms-plugin (ex. 5s). The default is 3 seconds.
 	// +optional
 	Timeout *metav1.Duration
+	// HealthzURL is the url on which kube-apiserver could check the health status of kms-plugin.
+	// +optional
+	HealthzURL string
+	// HealthzTimeout defines when to give-up on checking healthz.
```

Comment:
Hmm, i mis-interpreted this comment originally as "timeout for each healthz call attempt", not "timeout for all healthz calls combined, before stopping retries".
Maybe rephrase a bit to make it clearer?